2012-07-22  Thomas Schwinge  <thomas@codesourcery.com>

	* math/w_ilogb.c: Include <limits.h>.
	* math/w_ilogbl.c: Likewise.

---
 ChangeLog       |    5 +++++
 math/w_ilogb.c  |    2 +-
 math/w_ilogbf.c |    1 -
 math/w_ilogbl.c |    1 +
 4 files changed, 7 insertions(+), 2 deletions(-)

diff --git a/math/w_ilogb.c b/math/w_ilogb.c
index c87b517..7cb897a 100644
--- a/math/w_ilogb.c
+++ b/math/w_ilogb.c
@@ -18,6 +18,7 @@
 
 #include <math.h>
 #include <errno.h>
+#include <limits.h>
 #include <math_private.h>
 
 /* wrapper ilogb */
@@ -34,7 +35,6 @@ __ilogb (double x)
     }
   return r;
 }
-
 weak_alias (__ilogb, ilogb)
 #ifdef NO_LONG_DOUBLE
 strong_alias (__ilogb, __ilogbl)
diff --git a/math/w_ilogbl.c b/math/w_ilogbl.c
index 8c30caa..7cfc648 100644
--- a/math/w_ilogbl.c
+++ b/math/w_ilogbl.c
@@ -18,6 +18,7 @@
 
 #include <math.h>
 #include <errno.h>
+#include <limits.h>
 #include <math_private.h>
 
 /* wrapper ilogbl */
